### ISSUE TITLE
Finetuning Resnet-50 didn't get sharp result

### ISSUE THREAD

**Issue by xhwang** [iro-cp/FCRN-DepthPrediction#7]
Hi,
Thanks for sharing your excellent work. I try to finetune Resnet-50 with NYU-v2 labeled data (1449 pair) in caffe (intending to overfit the dataset).  

Augmentations are done as the paper describes, but depth uses **log** value. Upsamping layers are initialized with xavier. Batch_size: 1 (it costs 3G GPU memory),  lr: 1e-6 (the gradient will explode when using 1e-5).

The prediction somehow looks like ground truth depth, but loses details. Sample result:

![images](https://cloud.githubusercontent.com/assets/2776314/21168968/0b62147c-c1f3-11e6-9504-029d25605be2.png)

![depths](https://cloud.githubusercontent.com/assets/2776314/21168973/14f066ce-c1f3-11e6-89a2-ecef1d4beb8c.png)

When decrease learning rate, the loss did not decrease too much and the result depth map gets more blur rather than clearer. Do you have any ideas about this?

Sincerely

**Comment by iro-cp** [comment]
I cannot really answer with certainty regarding what goes wrong in your experiments, sorry. The whole setup is very different, including a re-implementation in caffe, fewer samples and the depth ground truth in log scale. All these could be possible issues. 
Also the batch size sounds rather strange as this model shouldn't need that much space. I could easily fit a batch of 8 images in a 4GB GPU during training, for example.

**Comment by xhwang** [comment]
@iro-cp  
Thanks for your reply. In the last two day, I did more experiments and got better result. As you said, using more data, or a more reliable loss function may achieve even better result.  
About the memory, it's my mistake. It does not cost that much.
Thanks again.

**Comment by Flying-Sailor** [comment]
@xhwang @iro-cp Hi, facing the similar issue of blurred predictions while training the model(even train images output is blurred ... )

**decoder**- random initialisation
**encoder** - fcrn pre-trained weights
**batchsize** - 10
**learning_rate** - 0.005
**momentum** - 0.9
**augmentation** - same as fcrn

 what did you change to get the better results.

**Comment by luohongcheng** [comment]
@Flying-Sailor Hi I am facing the same problem as you said. Have solved it?
